frontend/projects/upgrade/src/app/features/dashboard/profile/components/metrics/metrics.component.html

```diff
@@ -52,10 +52,10 @@
           <td class="ft-12-600" mat-cell *matCellDef="let element; let index = index">
             <ng-container *ngIf="element">
               <mat-tree [dataSource]="[element]" [treeControl]="nestedTreeControl" class="mat-nested-tree">
-                <!-- This is the tree node template for leaf nodes -->
+                <!-- Leaf node template -->
                 <mat-tree-node *matTreeNodeDef="let node">
-                  <button mat-icon-button>
-                    <mat-icon class="leaf-node-button-icon"> fiber_manual_record</mat-icon>
+                  <button mat-icon-button disabled>
```

Comment:
Does this have to be an always-disabled button? If it were just the plain icon it wouldn't be greyed-out, I think.

Reply:
The "disabled" state is added mainly to remove the hover effect (ripple) as it indicates the bullet is interactive when it's not. However, I think it makes sense to make them all black-colored for consistency as you suggested. I will apply this change.

Reply:
Just to be clear: I was asking why it needed to be a button at all. Can't it just be the icon, without having the <button> tags around it?

Reply:
Understood, thanks for clarifying. The button tags were already in place before I made any changes, so I'm not entirely sure of the original reason for using them. I'll go ahead and remove the button tags and keep just the icon.